### PR TITLE
subtract future fees

### PR DESCRIPTION
**Pull request by nandofw**
Subtract future fees if Subtract fee from amount is selected

image

one possible solve for #150

### CODE REVIEW

**Comment by npq7721** [comment]
have it been test with multiple recipients and inspect raw transaction to make sure it look right? @nandofw

**Comment by nandofw** [comment]
> have it been test with multiple recipients and inspect raw transaction to make sure it look right? @nandofw

I made some multiple transactions on 2 different wallets and the transactions appear to be correct

multi transaction  non modified wallet
[da621cc8d3666f14871ce75cac9b108b5ded4127b15478267af0ec353501f4c7](https://testnet.raptoreum.com/tx/da621cc8d3666f14871ce75cac9b108b5ded4127b15478267af0ec353501f4c7)

multi transaction modified wallet
[ffe39f8d5baa8ee02662563f4432e29e31c723f9c0967aaa1f2a42fe7774cda3](https://testnet.raptoreum.com/tx/ffe39f8d5baa8ee02662563f4432e29e31c723f9c0967aaa1f2a42fe7774cda3)

File: src/qt/sendcoinsentry.cpp [hunk 1]
```diff
@@ -204,7 +204,10 @@ SendCoinsRecipient SendCoinsEntry::getValue()
     // Normal payment
     recipient.address = ui->payTo->text();
     recipient.label = ui->addAsLabel->text();
-    recipient.amount = ui->payAmount->value();
+    CAmount amount = ui->payAmount->value();
+    if (ui->checkboxSubtractFeeFromAmount->isChecked() && ui->futureCb->isChecked())
+    amount -= getFutureFees() * COIN;
```
Comment (CodeIsTheKey) [inline src/qt/sendcoinsentry.cpp]:
```suggestion
       amount -= getFutureFeesCoin();
```

**Review by CodeIsTheKey — CHANGES_REQUESTED** [review]
This looks good to me.  All the testing I did looks good with single and multiple recipients, including on chain and in the wallet.  One minor cleanup in the change, otherwise looks good to me!

**Review by CodeIsTheKey — APPROVED** [review]

**Pull request merged**